smtsolver.js

```diff
@@ -1,38 +1,41 @@
-var commandExistsSync = require('command-exists').sync;
-var execSync = require('child_process').execSync;
-var fs = require('fs');
-var tmp = require('tmp');
+const commandExistsSync = require('command-exists').sync;
+const execSync = require('child_process').execSync;
+const fs = require('fs');
+const tmp = require('tmp');
 
+// Timeout in ms.
 const timeout = 10000;
 
-var potentialSolvers = [
+const potentialSolvers = [
   {
     name: 'z3',
+    command: 'z3',
     params: '-smt2 rlimit=20000000 rewriter.pull_cheap_ite=true fp.spacer.q3.use_qgen=true fp.spacer.mbqi=false fp.spacer.ground_pobs=false'
   },
+  {
+    name: 'Eldarica',
+    command: 'eld',
+    params: '-horn -t:' + (timeout / 1000)
+  },
```

Comment:
Just adding Eldarica for easy future use, it's currently not used for the tests.